### PR TITLE
JAMES-2172 Various posts on James website: Hactoberfest and Thibaut article

### PR DESCRIPTION
**Pull request by chibenwa**
capture d ecran de 2017-10-04 17-30-16

capture d ecran de 2017-10-04 17-34-04

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

File: src/homepage/_posts/2017-10-03-Hacktoberfest.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Hacktoberfest: contribute to James"
+date:   2017-10-03 16:13:22 +0200
+categories: james update
+---
+
+The James project joins the [Hactoberfest] !
```
Comment (strokyl) [inline src/homepage/_posts/2017-10-03-Hacktoberfest.markdown]: No space before !

File: src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+layout: post
+title:  "Blog post: Easy and secure James installation"
+date:   2017-10-03 16:13:22 +0200
+categories: james update
+---
+
+In a recent [blog post], Thibaut explain us how to easily set up a James sever on a personal domain.
+
+He gives hints about DNS configuration, SSL configuration, James configuration, and SPF configuration. All of
+it with working examples!
```
Comment (strokyl) [inline src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown]: I am not really good in english but are you sure "All of it with" is correct, for me it sound weird I would have written "All of this"

Comment (aduprat) [reply src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown]: _All those features explained with_

File: src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Blog post: Easy and secure James installation"
+date:   2017-10-03 16:13:22 +0200
+categories: james update
+---
+
+In a recent [blog post], Thibaut explain us how to easily set up a James sever on a personal domain.
```
Comment (mbaechler) [inline src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown]: without a link to a profile, nobody knows we is Thibaut, right ?

**Comment by chibenwa** [comment]
capture d ecran de 2017-10-06 10-12-11

capture d ecran de 2017-10-06 10-12-15

File: src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown (hunk 3 above)
Comment (thithib) [reply src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown]: "explain" -> "explains"
"sever" -> "server"

File: src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown (hunk 2 above)
Comment (thithib) [reply src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown]: I would say "TLS" instead of "SSL"

**Comment by chibenwa** [comment]
test this please

**Pull request merged**